packages/base/links-to-many-component.gts

```diff
@@ -107,24 +100,15 @@ class LinksToManyEditor extends GlimmerComponent<Signature> {
     }
   });
 
-  private createCard = restartableTask(async () => {
-    let cards = (this.args.model.value as any)[this.args.field.name];
-    let type = identifyCard(this.args.field.card) ?? baseCardRef;
-    let newCard: Card | undefined =
-      await this.args.context?.actions?.createCard(type, undefined, {
-        isLinkedCard: true,
-      });
-    if (newCard) {
-      cards = [...cards, newCard];
-      (this.args.model.value as any)[this.args.field.name] = cards;
-    }
-  });
-
   remove = (index: number) => {
     let cards = (this.args.model.value as any)[this.args.field.name];
     cards = cards.filter((_c: Card, i: number) => i !== index);
     (this.args.model.value as any)[this.args.field.name] = cards;
   };
+
+  getPluralDisplayName = () => {
```

Comment:
I agree with this. There is a module that I put in the host called `text-suggestion` which is meant to suggest the correct title text when choosing cards.

There is a function `titleText` there which has some vague notion of a checking a card and pluarizing the noun/object of a sentence. Perhaps you can extend this or even introduce a new module. 

The problem I see here is that the helper probably has to go into runtime-common to be shared across host and base since links-to-many-component is in base.

Reply:
I think we can use the same approach like this dompurify https://github.com/cardstack/boxel/blob/main/packages/runtime-common/dompurify.ts.

Reply:
I don't have strong opinions on this. The dompurify suggestion wud make the code available to most cards and services so I think it sounds good.